src/components/Popup.js

```diff
@@ -36,4 +38,12 @@ export default class Popup {
 
     this._popupElement.addEventListener("mousedown", this._handleClickOutside);
   }
+
+  renderloading(isLoading) {
```

Comment:
You shouldn't find a submit button in `Popup` because the image modal doesn't have it

And `renderloading` should not be in `Popup` due to the same issue